### DISCUSSION TITLE
Selbst erstellte firmware.bin

### DISCUSSION THREAD

**Discussion by jblond4711** [tbnobody/OpenDTU#2201] [Q&A]
Hi,

habe eben nach Anleitung eine Firmware.bin kompiliert. Mir kommt die *.bin mit 1.940.496 Kb ziemlich groß vor. In VSCode habe ich nur Github und die Platformio installiert und dann kompiliert.
Es ist nicht das Original sondern folgenden Fork habe ich genommen. [ https://github.com/SpaceInCat/OpenDTU_SDstorage](url)
Gibt es in VSCode noch einstellungen die vornehmen kann damit die erstellte .bin kleiner wird?

**Comment by stefan123t** [comment]
Evtl musst Du das Vue Frontend einmal erstellen. Das wird dann als gezipptes Archiv an den Browser ausgeliefert.
